hw3.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="hw3css.css">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300&family=Playfair+Display&display=swap"
+        rel="stylesheet">
+    <title>Домашка 3</title>
+</head>
+
+<body>
+    <div class="white_box">
+        <p class="willow">Willow Crest</p>
+        <div class="other_text">
+            <p class="fl"><span class="t">T</span>his happened, indeed, to the train in which Mr. Fogg was travelling.
+                About twelve o'clock a troop of ten or twelve thousand head of buffalo encumbered the track. The
+                locomotive, slackening its speed, tried to clear the way with its cow–catcher; but the mass of animals
+                was too great. The buffaloes marched along with a tranquil gait, uttering now and then deafening
+                bellowings. There was no use of interrupting them, for, having taken a particular direction, nothing can
+                moderate and change their course; it is a torrent of living flesh which no dam could contain.
+            </p>
+            <p class="text">The travellers gazed on this curious spectacle from the platforms; but Phileas Fogg, who
+                had the most reason of all to be in a hurry, remained in his seat, and waited philosophically until
+                it should please the buffaloes to get out of the way.
+                Passepartout was furious at the delay they occasioned, and longed to discharge his arsenal of
+                revolvers upon them.</p>
+            <p class="text">"What a country!" cried he. "Mere cattle stop the trains, and go by in a procession,
+                just as if they were not impeding travel! Parbleu! I should like to know if Mr. Fogg foresaw this
+                mishap in his programme! And here's an engineer who doesn't dare to run the locomotive into this
+                herd of beasts!"</p>
+        </div>
+        <div>
+            <p class="engineer"><span class="rystochka">''</span><span class="indent">The engineer did not try to
```

Comment:
wtf is 'rystochka'? why not - 'huita' or 'ebany-kovichki'? don't use Translit pls :)